composer.json

```diff
@@ -50,6 +50,7 @@
         "mexitek/phpcolors": "^1.0",
         "monolog/monolog": "^3.5",
         "paragonie/sodium_compat": "^1.20",
+        "phpoffice/phpspreadsheet": "^1.29",
```

Comment:
A 2.0.0 version of `phpoffice/phpspreadsheet` that has been released a month ago. According to the release notes, migrating to it should be painless. Could you try to upgrade it?

Reply:
Oh, I did not know there were a new release, I'll upgrade, yes.

Reply:
It's done